macros/src/interface/generate.rs

```diff
@@ -72,17 +74,6 @@ fn generate_client(item_tokens: &mut TokenStream, fizyr_rpc: &syn::Ident, interf
 				Self { peer }
 			}
 
-			/// Connect to a remote server.
-			///
-			/// See [`fizyr_rpc::Peer::connect`](https://docs.rs/fizyr-rpc/latest/fizyr_rpc/struct.Peer.html#method.connect) for more details.
-			pub async fn connect<'a, Transport, Address>(address: Address, config: Transport::Config) -> std::io::Result<Self>
-			where
-				Address: 'a,
-				Transport: #fizyr_rpc::transport::Transport<Body = F::Body> + #fizyr_rpc::util::Connect<'a, Address>,
-			{
-				Ok(#fizyr_rpc::Peer::<Transport>::connect(address, config).await?.into())
-			}
-
```

Comment:
Why is this removed?

Reply:
Because it's complicated to use correctly. You would always need to manually fill in the `Transport` and `Address` type when calling it, which also requires you to fill in the lifetime.

I actually discussed this with you, and we decided to drop the `connect()` function :p